identity-vault/getting-started/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "getting-started",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build && cap sync",
+    "preview": "vite preview",
+    "test.e2e": "cypress run",
+    "test.unit": "vitest",
+    "lint": "eslint"
```

Comment:
We should include the other scripts that the other projects have in their `package.json` file.